src/commands/aksAccount/aksAccount.ts

```diff
@@ -50,18 +49,12 @@ export async function selectSubscriptions(): Promise<void> {
 
     const allSubscriptions = await getSubscriptions(sessionProvider.result, SelectionType.All);
     if (failed(allSubscriptions)) {
-        window.showErrorMessage(allSubscriptions.error);
+        await window.showErrorMessage(allSubscriptions.error);
         return;
     }
 
     if (allSubscriptions.result.length === 0) {
-        const noSubscriptionsFound = "No subscriptions were found. Set up your account if you have yet to do so.";
-        const setupAccount = "Set up Account";
-        const response = await window.showInformationMessage(noSubscriptionsFound, setupAccount);
-        if (response === setupAccount) {
-            env.openExternal(Uri.parse("https://azure.microsoft.com/"));
-        }
-
+        await handleNoSubscriptionsFound();
```

Comment:
💡Note:  this function is `await` but returns `void` we need to be careful of this, `promise<void>` means there is no check for this, lets see if we can return the type of `result` there is various usage across code and we can discuss, (won't take too much of time)

Reply:
Good idea. Removed the await and just return void, since we only display a vscode notification.